wilma-mock/build.gradle

```diff
@@ -0,0 +1,69 @@
+/*==========================================================================
+Copyright 2015 EPAM Systems
+
+This file is part of Wilma.
+
+Wilma is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Wilma is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Wilma.  If not, see <http://www.gnu.org/licenses/>.
+===========================================================================*/
+
+description = "wilma-mock"
+
+buildscript {
+    repositories { jcenter() }
+    dependencies {
+        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.0'
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'application'
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
no need for plugin shadow, as shadow need only if we create jar that contains all others jars this module will depend
it is enough if 'gradlew jar' target works and creates the jar itself.